metadata.yaml

```
@@ -0,0 +1,10 @@
+name: glauth-operator
+display-name: glauth-operator
```

Should the display name just be `GLAuth Operator`?

```suggestion
display-name: GLAuth Operator
```

---

Sounds good, updated!